backfill/Cargo.toml

```diff
@@ -22,7 +22,7 @@ tokio.workspace = true
 
 # rpc
 async-trait = "0.1"
-jsonrpsee = { version = "0.23", features = ["server", "macros"] }
+jsonrpsee = { version = "0.24.2", features = ["server", "macros"] }
```

Comment:
for some reason, there was also a version mismatch for jsonrpsee. this version bump fixes it.